commit 4bae2468ddd53cf5cea9a2f38b87537c9ba2f0c3
Author: Isaac Alejandro Ortiz Zaldivar <a24216345@alumnos.uady.mx>
Date:   Sat May 31 02:57:26 2025 -0600

    feat: Add changelog and update repository structure
    
    - Introduced detailed changelog for the "Batalla Naval" project, summarizing recent updates and features.
    - Created a repository changelog to document structural changes and configurations.
    - Added new files for diffs and updated README with project index and navigation.
    - Refactored the changelog generation script to point to the correct location.
    - Included visual assets and documentation in the project structure for better clarity and organization.

diff --git a/codigo_fuente/flow_chart.md b/codigo_fuente/flow_chart.md
deleted file mode 100644
index 79d02cc..0000000
--- a/codigo_fuente/flow_chart.md
+++ /dev/null
@@ -1,31 +0,0 @@
-``` mermaid
-flowchart TD
-Pantalla_bienvenida --Limpiar pantalla()--> menu_principal 
-menu_principal ---->partida(["partida()"])
-menu_principal --limpiar_pantalla()-->Reglas(["reglas()"])
-menu_principal--limpiar_pantalla()-->Mensaje_salida([Saliendo del programa])
-
-%% Reglas
-Reglas-->Mostrar_reglas--> Opc{"Press z to return main"}
-Opc --Any other than z--> Reglas
-Opc --Pressing z-->menu_principal
-
-%% Partida
-partida --limpiar_pantalla()-->mpt(["menu_por_turno()"])
-
-
-%% Menu por turno
-mpt ---->carta(["sacar_carta()"])
-mpt -->rptB(["mostrar_reporte_barcos"])
-mpt -->rptBE(["mostrar_reporte_barcos_enemigos"])
-
-%% reporte barcos
-rptB -->OpcB{"Press z to return main"}
-OpcB --Any other than z--> rptB
-OpcB --Pressing z-->mpt
-
-%% Reporte barcos enemigos
-rptBE -->OpcBE{"Press z to return main"}
-OpcBE --Any other than z--> rptBE
-OpcBE --Pressing z-->mpt
-```
\ No newline at end of file
